addon/components/cfb-form-editor/question/options.js

```diff
@@ -83,6 +100,51 @@ export default RenderComponent.extend({
     this.setDirty();
   },
 
+  setupUIkit() {
+    UIkit.util.on("#option-list", "moved", (...args) =>
+      run(this, this._handleMoved, ...args)
+    );
+  },
+
+  _handleMoved({ detail: [sortable] }) {
+    const options = [...sortable.$el.children];
+    // Remove last element as it is the add row button
+    options.pop();
```

Comment:
```suggestion
    // Remove last element as it is the add row button
    const options = [...sortable.$el.children].slice(0, -1);
```
Is easier to understand and doesn't mutate the `options`

Reply:
Changed